client/src/components/Grid/GridList.vue

```diff
@@ -0,0 +1,283 @@
+<script setup lang="ts">
+import { BAlert, BButton, BLink, BPagination } from "bootstrap-vue";
+import { computed, onMounted, ref, watch } from "vue";
+import { useRouter } from "vue-router/composables";
+
+import { timeout } from "@/utils/timeout";
+
+import { Config, FieldKeyHandler, Operation, RowData } from "./configs/types";
+
+import GridLink from "./GridElements/GridLink.vue";
+import GridOperations from "./GridElements/GridOperations.vue";
+import GridText from "./GridElements/GridText.vue";
+import FilterMenu from "@/components/Common/FilterMenu.vue";
+import SharingIndicators from "@/components/Indices/SharingIndicators.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+import StatelessTags from "@/components/TagsMultiselect/StatelessTags.vue";
+//@ts-ignore
```

Comment:
When adding a ts-ignore, please state why behind it.
Removing the ts-ignore here has no effect for me. Could this be outdated, or a faulty local config?